contrib/occtax/data/occtax.sql

```diff
@@ -886,10 +880,13 @@ BEGIN
       id_nomenclature_geo_object_nature = NEW.id_nomenclature_geo_object_nature,
       last_action = 'U',
       comment_context = NEW.comment,
-      additional_data = NEW.additional_fields || occurrence.additional_fields || counting.additional_fields
+      additional_data = additional_data || NEW.additional_fields
```

Comment:
D'ou vient la variable `additional_data` ? C'est l'ancienne valeur du champs `additionnal_data` ?
tu coup tu concatenne l'ancienne à la nouvelle ?

Reply:
`additional_data` vient de la table `synthese`
`||` permet de faire un update d'un `jsonb`
https://stackoverflow.com/questions/26703476/how-to-perform-update-operations-on-columns-of-type-jsonb-in-postgres-9-4